app/ts/components/simulationExplaining/identifyTransaction.tsx

```diff
@@ -85,13 +90,32 @@ export function identifyGovernanceVote(simTx: SimulatedAndVisualizedTransaction)
 	const fourByte = get4Byte(simTx.transaction.input)
 	if (fourByte === undefined) return undefined
 	const explanation = FourByteExplanations[fourByte]
-	if (explanation !== 'Cast vote' && explanation !== 'Submit vote') return undefined
-	return {
-		type: 'GovernanceVote' as const,
-		title: `Governance Vote`,
-		signingAction: `Submit Vote`,
-		simulationAction: `Simulate Vote`,
-		rejectAction: `Reject Vote`,
+	if (explanation !== 'Cast Vote'
+		&& explanation !== 'Submit Vote'
+		&& explanation !== 'Cast Vote By Sig'
+		&& explanation !== 'Cast Vote With Reason'
+		&& explanation !== 'Cast Vote With Reason And Params'
+		&& explanation !== 'Cast Vote With Reason And Params By Sig'
```

Comment:
The changes I proposed to these strings *should* result in a compiler error here.  If not, we should make necessary changes to ensure it does.  Perhaps by using a switch instead?